src/NServiceBus.AcceptanceTesting/Support/ScenarioConfigSource.cs

```diff
@@ -70,15 +70,6 @@ public T GetConfiguration<T>() where T : class, new()
                 }
             }
 
-            if (type == typeof(Logging))
```

Comment:
I think you are right. I was concerned since the fault config part was called. I just double checked it and it never hit my breakpoint inside this method